src/commands/NewIntegrationFileCommand.ts

```diff
@@ -33,9 +33,10 @@ const LANGUAGES_WITH_FILENAME_EXTENSIONS = new Map([
 const LANGUAGES = Array.from(LANGUAGES_WITH_FILENAME_EXTENSIONS.keys());
 
 export async function create() : Promise<void> {
-	const language = await vscode.window.showQuickPick(LANGUAGES);
+	const language = await vscode.window.showQuickPick(LANGUAGES, {placeHolder:'Please select the language in which the new file will be generated.'});
 	if (language) {
-		const workspaceFolder = await vscode.window.showWorkspaceFolderPick();
+		const workspaceFolder = await vscode.window.showWorkspaceFolderPick(
+			{placeHolder: 'Please select the Workspace folder in which the new file will be created.'});
```

Comment:
I think workspace should not start with capital letter. Would you agree?

Reply:
in the UI and documentation there is a mix of capital letters and not capital letters.
At least needs to be either "Workspace Folder" or "workspace folder". I changed to "workspace folder"